extractor/src/main/java/org/schabi/newpipe/extractor/services/peertube/PeertubeService.java

```diff
@@ -77,7 +77,10 @@ public SubscriptionExtractor getSubscriptionExtractor() {
     @Override
     public ChannelExtractor getChannelExtractor(ListLinkHandler linkHandler)
             throws ExtractionException {
-        return new PeertubeChannelExtractor(this, linkHandler);
+
+        return linkHandler.getUrl().matches("^.*\\/accounts\\/[^\\/]*$") ?
+                new PeertubeUserExtractor(this, linkHandler) :
+                new PeertubeChannelExtractor(this, linkHandler);
```

Comment:
Please change it to:

```suggestion
        if (linkHandler.getUrl().contains("/video-channels/")) {
            return new PeertubeChannelExtractor(this, linkHandler);
        } else {
            return new PeertubeUserExtractor(this, linkHandler);
        }
```